pages/index.tsx

```diff
@@ -5,10 +5,10 @@ import About from '@/pages/sections/about';
 import Tracks from '@/pages/sections/tracks';
 import Support from '@/pages/sections/support';
 import Sponsors from '@/pages/sections/sponsors';
-import Staff from '@/pages/sections/staff';
 import Faq from '@/pages/sections/faq';
 import Schedule from '@/pages/sections/schedule';
 import React from 'react';
+import Passports from '@/components/Passports.tsx';
```

Comment:
vercel is failing deployment because of this import statement right here
should be
```suggestion
import Passports from '@/components/Passports';
```